src/geovista/examples/contour/banded_contour.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021, GeoVista Contributors.
+#
+# This file is part of GeoVista and is distributed under the 3-Clause BSD license.
+# See the LICENSE file in the package root directory for licensing details.
+
+"""
+Banded Contour
+--------------
+
+This example demonstrates how to use banded contours.
+
+📋 Summary
+^^^^^^^^^^
+Uses an unstructured Met Office LFRic C48 cubed-sphere mesh of surface altitude
+data.
+
+First triangulate the mesh as this is the preferred geometry when using the contour
+filter.
+
+Then we threshold the surface altitude by height such that all sea-level cells
+are removed from the mesh.
+
+The remaining land-based cells of the mesh are contoured and rendered using a
+qualitative colormap (``Set3``) containing 12 unique colours. We use the
+``contour_banded`` filter to generate 13 contours and the associated banded contour
+edges.
+
+A Natural Earth base layer is also rendered along with the Natural Earth coastlines.
+
+.. tags::
+
+    component: coastlines,
+    component: texture,
+    domain: orography,
+    filter: cast,
+    filter: contour,
+    filter: threshold,
+    sample: unstructured,
+    style: colormap
```

Comment:
```suggestion
    component: coastlines, component: texture,
    domain: orography,
    filter: cast, filter: contour, filter: threshold, filter: triangulate,
    sample: unstructured,
    style: colormap
```

Reply:
@HGWright Forgot to mention that we can have multi-line `sphinx-tags`, which makes things a tad tidier :+1: 

Also added the `filter: triangulate` tag